sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslSqlStdOperatorsTest.java

```diff
@@ -311,7 +312,7 @@ public void testArithmeticOperator() {
             .addExpr("c_double + c_bigint", 2.0)
             .addExpr("1 - 1", 0)
             .addExpr("1.0 - 1", ZERO_1)
-            .addExpr("1 - 0.0", ONE_0)
+            .addExpr("1 - 0.0", ONE_1)
```

Comment:
Why do we end up needing these test changes? Is there a small behavior difference here?

Reply:
updated CHANGES.md to mention these kind of changes